### ISSUE TITLE
\ensuremath

### ISSUE THREAD

**Issue by nschloe** [KaTeX/KaTeX#3687]
I'd love for `\ensuremath` to work. It can really just be ignored in KaTeX, and it helps when translating things from proper LaTeX documents.

**Comment by edemaine** [comment]
Agreed. This would be easy to add once we have `\ifmmode` via #3385. Then `\def\ensuremath#1{\ifmmode#1\else$#1$\fi}` should work. Perhaps it's time for me to revise #3385.
